backend/de.metas.business/src/main/java/de/metas/invoice/service/impl/InvoiceDAO.java

```diff
@@ -53,6 +62,33 @@ public I_C_Invoice createInvoice(String trxName)
 		return InterfaceWrapperHelper.create(Env.getCtx(), I_C_Invoice.class, trxName);
 	}
 
+	@Override
+	public Map<OrderId, InvoiceId> getInvoiceIdsForOrderIds(final List<OrderId> orderIds)
```

Comment:
move this method into AbstractInvoiceDAO. Is has no hardcoded SQL-stuff, so we don't need a dedicated `PlainInvoiceDAO.java` implementation